bluebottle/members/serializers.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = BB_USER_MODEL
-        fields = ('id', 'first_name', 'last_name', 'username',
```

Comment:
In some parts of Reef we still rely on usernames, e.g.
https://github.com/onepercentclub/reef/blob/master/frontend/app/controllers/task.js#L44-L54
We should fix that before we deploy this.